### PR TITLE
tectonic-installer/config.tf: Add a link to the doc describing how to generate bcyp-encoded hash of a password

### PR DESCRIPTION
**Pull request by radhikapc**
ref: https://github.com/coreos-inc/tectonic-issues/issues/245

### CODE REVIEW

**Comment by radhikapc** [comment]
@robszumski @coresolve plz approve if everything looks fine.

**Comment by aknuds1** [comment]
I agree that docs should be linked to for this :) Might it be a better idea to link to the bcrypt-tool project root though, rather than the releases page? I'm thinking it would generally be a better idea to see the main README first, as a way of documenting how to use the tool, which *should* link to a downloads page.

File: config.tf [hunk 1]
```diff
@@ -196,7 +196,7 @@ variable "tectonic_admin_email" {
 
 variable "tectonic_admin_password_hash" {
   type        = "string"
-  description = "bcrypt hash of admin password to use with Tectonic Console"
+  description = "bcrypt hash of admin password to use with Tectonic Console. Generate a bcrypt hash of the password by using the tool given at https://github.com/coreos/bcrypt-tool/releases."
```
Comment (aknuds1) [inline config.tf]: How about linking to https://github.com/coreos/bcrypt-tool instead, as this should be the main documentation page for bcrypt-tool?

Comment (radhikapc) [inline config.tf]: ok. i will update the PR

**Comment by s-urbaniak** [comment]
I have addressed this already in #536.

**Comment by s-urbaniak** [comment]
Closing in favor of https://github.com/coreos/tectonic-installer/pull/536

**Pull request closed without merging**